### PR TITLE
feat: update address description

### PR DESCRIPTION
**Pull request by classicalliu**

### CODE REVIEW

File: packages/neuron-wallet/src/controllers/wallets/index.ts [hunk 1]
```diff
@@ -331,7 +331,13 @@ export default class WalletsController {
 
   @CatchControllerError
   public static async updateAddressDescription({ address, description }: { address: string; description: string }) {
-    // TODO: update description of specified address
+    const walletService = WalletsService.getInstance()
+    const wallet = walletService.getCurrent()
```
Comment (ashchan) [inline packages/neuron-wallet/src/controllers/wallets/index.ts]: Please refactor that: 1) wallet id is always passed to controllers or other wallet layers from UI, 2) `neuron-wallet` uses `walletService.getCurrent()` as few as possible.

Comment (classicalliu) [reply packages/neuron-wallet/src/controllers/wallets/index.ts]: updated

File: packages/neuron-wallet/src/controllers/wallets/index.ts [hunk 2]
```diff
@@ -330,12 +330,24 @@ export default class WalletsController {
   }
 
   @CatchControllerError
-  public static async updateAddressDescription({ address, description }: { address: string; description: string }) {
-    // TODO: update description of specified address
+  public static async updateAddressDescription({
+    id,
```
Comment (ashchan) [inline packages/neuron-wallet/src/controllers/wallets/index.ts]:
```suggestion
    walletId,
```

File: packages/neuron-wallet/src/controllers/wallets/index.ts [hunk 3]
```diff
@@ -330,12 +330,24 @@ export default class WalletsController {
...
+    address,
+    description,
+  }: {
+    id: string
```
Comment (ashchan) [inline packages/neuron-wallet/src/controllers/wallets/index.ts]:
```suggestion
    walletId: string
```

File: packages/neuron-wallet/src/controllers/wallets/index.ts [hunk 4]
```diff
@@ -330,12 +330,24 @@ export default class WalletsController {
...
+    address: string
+    description: string
+  }) {
+    const walletService = WalletsService.getInstance()
+    const wallet = walletService.get(id)
```
Comment (ashchan) [inline packages/neuron-wallet/src/controllers/wallets/index.ts]:
```suggestion
    const wallet = walletService.get(walletId)
```

File: packages/neuron-wallet/src/controllers/wallets/index.ts [hunk 5]
```diff
@@ -330,12 +330,24 @@ export default class WalletsController {
...
+
+    await AddressService.updateDescription(wallet.id, address, description)
 
     return {
       status: ResponseCode.Success,
       result: {
+        id,
```
Comment (ashchan) [inline packages/neuron-wallet/src/controllers/wallets/index.ts]:
```suggestion
        walletId,
```

**Pull request merged**